whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -12744,45 +12745,75 @@
       "$6": {"property": "marc:fieldref"}
     },
     "533": {
-      "addLink": "hasReproduction",
-      "embedded": true,
-      "resourceType": "Instance",
       "pendingResources": {
-        "_:event": {"addLink": "provisionActivity", "resourceType": "ProvisionActivity"}
+        "_:event": {"addLink": "production", "resourceType": "Reproduction"}
```

Comment:
Take care when Reproduction is declared as a subclass of Production, so that these don't revert to bib 264 instead (adding 533 to `onRevertPrefer` of 264 ought to be enough).

Test drive this using locally built definitions, which are scooped up by: https://github.com/libris/librisxl/blob/0e368b26378332ee0e9eed14e1277d85330b9168/whelk-core/src/integTest/groovy/MarcFrameConverterSpec.groovy#L29 using: https://github.com/libris/librisxl/blob/0e368b26378332ee0e9eed14e1277d85330b9168/whelk-core/src/main/groovy/whelk/converter/marc/MarcFrameCli.groovy#L72-L73

Reply:
They were needed on 257 and 261 as well, I don't think the onRevertPrefer is transitive?

Reply:
And yes the testing was done with freshly built vocab :)